powsybl-build-tools/src/main/resources/powsybl-build-tools/lombok.config

```diff
@@ -0,0 +1,14 @@
+# Don't search another lombok.config file in parent folders
+config.stopbubbling = true
+
+# We add annotations flags for IDEs and tools :
+#  * mark the code as generated for linters and coverage tools
+lombok.addJavaxGeneratedAnnotation = true
```

Comment:
Do we want this ? I see in the docs:
" BREAKING CHANGE: Starting with lombok v1.16.20, defaults to false instead of true, as this annotation is broken in JDK9."

Reply:
I think it's with jigsaw/modules system that it broke, but we don't use it. _See (lombok#985)[https://github.com/projectlombok/lombok/issues/985]. _  
After did we consider that some projects use/will use it?